src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final String[] FIGURE_COLOR = new String[] {"blue", "yellow", "green", "black",

```

Comment:
```suggestion
    public static final String[] figureColor = new String[] {"blue", "yellow", "green", "black",

```
думаю тут більше такий неймінг підійде так як хоч змінна і статична + фінальна, елементи масива можна змінювати за індексом